client/index.html

```
@@ -218,38 +218,20 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
-									<input type="checkbox" name="join">
-									Show joins
+									<input type="checkbox" name="types">
+									Show <abbr title="Joins, parts, kicks, nick changes, and mode changes">status messages</abbr>
```

Either this needs to be checked by default, or it has to be `Hide`.